smarts/core/smarts.py

```diff
@@ -80,7 +79,7 @@ class SMARTS:
     def __init__(
         self,
         agent_interfaces,
-        traffic_sim: SumoTrafficSimulation,
+        traffic_sim,  # SumoTrafficSimulation
```

Comment:
I am not sure we need to specify that it is `SumoTrafficSimulation` any more since we are getting away from that.

Reply:
Yeah, agreed.  But I'll remove that in a future PR.